docs/modules/data-structures/pages/vector-search-overview.adoc

```diff
@@ -43,17 +43,26 @@ The index is based on the link:https://github.com/jbellis/jvector[JVector] libra
 
 Each collection is partitioned and replicated based on the system's general partitioning rules. Data partitioning is carried out using the collection key.
 
-For further information on Hazelcast partitioning, see xref:architecture:data-partitioning.adoc[Data Partitioning and Replication].
+Vector collection is AP data structure and implements standard xref:data-structures:backing-up-maps.adoc#in-memory-backup-types[in-memory backup types].
```

Comment:
Will users know what AP means or should we spell it out?

Reply:
added link: https://github.com/hazelcast/hz-docs/pull/1366/commits/18692302ac5c19523788d15679a6fbdf3ae05bbb